utils.js

```diff
@@ -25,9 +25,15 @@ const makePromise = () => {
   p.reject = reject;
   return p;
 };
+
+function timeout(ms, result = true) {
```

Comment:
The second argument is not used here.

Also it confuses me why it would ever be needed, since the caller has a closure.

Reply:
This was directly ripped from my own library code, the second argument is an optional value for resolve(). It's not needed in this one use case but it was quicker to just paste the function and move on, leaving optional functionality intact.

Reply:
Seems like we are trashing the codebase then.

Reply:
I'll remove it. Since I use this method often when working with async code, I figured it would get wrapped into library code and serve more use in case someone needs timeout to resolve to something. The default of `true` means that timeout can be used as such: `await timeout(100) && doThing()` which is idiomatic and useful.